components/engine/src/errors.rs

```diff
@@ -1,5 +1,8 @@
 // Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
 
+#[cfg(feature = "prost-codec")]
+use prost;
+use protobuf::ProtobufError;
```

Comment:
em, do we need to use protobuf feature here? 

seem if we pass prost-codec feature, we still need to import protobuf?

Reply:
I use the `protobuf::Message` trait (and supporting error types) to support both Prost and rust-proto, so we still need the protobuf imports in the Prost case.

Reply:
got it